packages/perseus/src/widgets/__stories__/interactive-graph-regression.stories.tsx

```diff
@@ -157,6 +157,20 @@ export const MafsWithMultipleSegments = (
     />
 );
 
+export const MafsCircleGraphWithNonsquareRange = (
```

Comment:
Question: Was this just a specific case that we want to keep an eye on? The graph definitely looks unusual. Is something like this in production?

Reply:
I don't think we have any examples like this in production, but since the old graph supports this case, I wanted to make sure we don't regress.